src/client/src/components/pages/participants/addParticipantCourse/staticData.js

```diff
@@ -0,0 +1,56 @@
+export const state = {
+    courseName: "",
+    startDate: "",
+    endDate: ""
+};
+export const fields = [
+    [{
+        tag: "DropDown",
+        label: "Course Name",
+        name: "courseName",
+        width: "40rem",
+        options: [  'Level 2 Diploma in Scaffolding', 
+                    'Level 1 Certificate in Construction Skills (Painting & Decorating)',
+                    'Level 2 Certificate in Preparation and Application of Paint Systems',
+                    'Level 2 Diploma in Dry Lining',
+                    'Level 1 Award in Construction Skills (Painting and Decorating)',
+                    'Level 1 Award in  Employability',
+                    'CSCS Card',
+                ]
+    }
+    ],
+    [
+        {
+            tag: "Input",
+            label: "Course start",
+            name: "startDate",
+            width: "20rem",
+            type: "date",
+        },
+        {
+            tag: "Input",
+            label: "Course end",
+            name: "endDate",
+            width: "20rem",
+            type: "date",
+        }
+    ],
+    [
+        {
+        tag: 'Button',
+        value: "Add Course",
+        color: "#272727",
+    },
```

Comment:
You need to add `staticField: true` to make the buttons appear in your form

Reply:
it was appeared in my form, 
there is no need of use it in my pages

Reply:
I solve it now

Reply:
:+1: